test-cases/manager/manager-regression-multiDC-gce.yaml

```diff
@@ -1,7 +1,7 @@
 test_duration: 360
 
-stress_cmd: "cassandra-stress write cl=QUORUM n=4000000 -schema 'replication(strategy=NetworkTopologyStrategy,us-east1scylla_node_east=2,us-west1scylla_node_west=1)' -mode cql3 native -rate threads=200 -pop seq=400000000..600000000"
-stress_read_cmd: "cassandra-stress read cl=QUORUM n=4000000 -schema 'replication(strategy=NetworkTopologyStrategy,us-east1scylla_node_east=2,us-west1scylla_node_west=1)' -mode cql3 native -rate threads=200 -pop seq=400000000..600000000"
+stress_cmd: "cassandra-stress write cl=LOCAL_QUORUM n=4000000 -schema 'replication(strategy=NetworkTopologyStrategy,us-east1scylla_node_east=2,us-west1scylla_node_west=1)' -mode cql3 native -rate threads=200 -pop seq=400000000..600000000"
```

Comment:
The same as in the `internal_test_data/multi_region_dc_test_case.yaml`

Reply:
@juliayakovlev since we are splitting the loaders in this case, there's no real reason to use  `LOCAL_QUORUM`